pom.xml

```diff
@@ -23,7 +23,7 @@
     <yorc.google.types.version>1.0.0</yorc.google.types.version>
     <yorc.hp.types.version>1.0.0</yorc.hp.types.version>
     <yorc.k8s.types.version>2.0.0-SNAPSHOT</yorc.k8s.types.version>
-    <yorc.os.types.version>1.1.0</yorc.os.types.version>
+    <yorc.os.types.version>1.2.0</yorc.os.types.version>
```

Comment:
Need to pass in SNAPSHOT os and slurm types

Reply:
ok, merged this branch with develop branch which has already this change after your last push.